docs/DeveloperGuide.md

```diff
@@ -473,7 +455,7 @@ The following sequence diagram illustrates how the `add` command works:
 The lifeline for `AddCommandParser` should end at the destroy marker (X) but due to a limitation of PlantUML, the lifeline reaches the end of diagram.
 </div>
 
-The following activity diagram shows how the `add` command works:
+The following activity diagram shows what happens when a `user` execute the `add` command.
```

Comment:
wait is there a need for user to be `user` instead of user

Reply:
oh yes, my mistake will fix it thanks